airbyte-integrations/base-python/base.py

```diff
@@ -86,7 +86,7 @@ def start(self):
                 # todo: pass in state
                 generator = source.read(logging, rendered_config_path)
                 for message in generator:
-                    print(message.message_string.strip())
+                    print(message.serialize(sort_keys=True))
```

Comment:
why do we sort keys? is this needed or just a nice to have? could you mention this context in a comment?

Reply:
removed